doc/interfaces/reconstruction_flow.rst

```diff
@@ -0,0 +1,269 @@
+                       D_{zx} & D_{zy} & D_{zz} \\ \end{pmatrix}
+
+This matrix is a variance/covariance matrix of the diffusivity along the three
+spatial dimensions. Note that we can assume that diffusivity has antipodal
+symmetry, so elements across the diagonal are equal. For example:
+$D_{xy} = D_{yx}$. This is why there are only 6 free parameters to estimate
+here.
+
+We will use the ``stanford_hardi`` dataset in DIPY to showcase this reconstruction
+method. As with any other workflow in DIPY, you can also use your own data!
+
+Create a directory in which to save the output volumes(e.g.:
+``recons_dti_output``)::
+    
+    mkdir recons_dti_output
+
+To run the Diffusion Tensor Imaging reconstruction method, we need to specify the 
+paths to the diffusion input file, b-values file, b-vectors file and mask file,
+followed by optional arguments. In this case, we will be specifying the list of
+metrics to save (``save_metrics``), the output directory (``out_dir``), and the
+name of the tensors volume to be saved (``out_tensor``).
+
+We will run the command as::
+
+    dipy_fit_dti data/stanford_hardi/HARDI150.nii.gz data/stanford_hardi/HARDI150.bval data/stanford_hardi/HARDI150.bvec stanford_hardi_mask/brain_mask.nii.gz --save_metrics "md" "mode" "tensor" --out_dir "recons_dti_output" --out_tensor "dti_tensors.nii.gz"
+
+This command will save the DTI metrics to the specified output directory.
+
+--------------------------
+Diffusion Kurtosis Imaging
+--------------------------
+
+The diffusion kurtosis model is an expansion of the diffusion tensor model. In
+addition to the diffusion tensor (DT), the diffusion kurtosis model quantifies
+the degree to which water diffusion in biological tissues is non-Gaussian using
+the kurtosis tensor (KT) [Jensen2005]_.
+
+Measurements of non-Gaussian diffusion from the diffusion kurtosis model are of
+interest because they can be used to charaterize tissue microstructural
+heterogeneity [Jensen2010]_. 
+
+Moreover, DKI can be used to:
+
+* Derive concrete biophysical parameters, such as the density of axonal fibers and diffusion tortuosity [Fierem2011]_
+
+* Resolve crossing fibers in tractography and to obtain invariant rotational measures not limited to well-aligned fiber populations [NetoHe2015]_.
+
+We will use the ``cfin_multib`` dataset in DIPY to showcase this reconstruction
+method. You can also use your own data!
+
+Create the directories in which to save the DKI metrics (e.g.: ``recons_dki_output``)
+and mask file (e.g.: ``cfin_multib_mask``)::
+
+    mkdir recons_dki_output cfin_multib_mask
+
+The Diffusion Kurtosis Imaging reconstruction method requires the paths to the
+diffusion input file, b-values file, b-vectors file and mask file. The optional
+parameters can also be provided. In this case, we will be specifying the threshold
+used to find b=0 directions (``b0_threshold``) and the output directory (``out_dir``). 
+
+To get the mask file, run the command as::
```

Comment:
Same comment for the median Otsu method as above.

Reply:
Addressed in 4ce067c